File: Hilos Java/Ficheros texto_binario/EscribirArchivo.java
import java.io.IOException;
import java.io.PrintWriter;

public class EscribirArchivo {

    public static void main(String[] args) {
        PrintWriter escritor = null;

        try{
            escritor = new PrintWriter("archivo.txt");
        
            escritor.println("Esto es una linea del fichero");
            escritor.write("Esto es una linea del fichero");
            System.out.println("El archivo ha sido creado y escrito");
            escritor.close();
        }catch(IOException e){

        }
    }
}

File: Ficheros texto_binario/EscribirArchivo.py
try:
    with open("archivo.txt", "w") as escritor:
        escritor.write("Esto es una linea del fichero\n")  # Equivalente a println en Java
        escritor.write("Esto es una linea del fichero")  # Equivalente a write en Java
    print("El archivo ha sido creado y escrito")
except IOError as e:
    print(f"Error al escribir el archivo: {e}")
